offchain/Cargo.lock

```diff
@@ -18,7 +18,7 @@ version = "0.5.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "617a8268e3537fe1d8c9ead925fca49ef6400927ee7bc26750e90ecee14ce4b8"
 dependencies = [
- "bitflags",
+ "bitflags 1.3.2",
```

Comment:
We don't need to update the whole Cargo.lock. You can reset your changes, perform a cargo build, and then commit the changes. I did that in my PR: https://github.com/cartesi/rollups/pull/146/commits/3a99325244ce914b7aa54898362b5b3dfe7bb651

Reply:
Done at 06e89c7e